docs/content/components/collection/table/table.mdx

```diff
@@ -76,6 +76,12 @@ The `core-Theme` has also two variants called `tableLines` and `tableBorder`.
 
 <ComponentDemo file="./table-compact.demo.tsx" />
 
+### Aligned Columns
+
+In this example you can align any Table Column by using `align` property over `<Table.Column align='right'>` .
```

Comment:
can you also add the new prop in the `<PropsTable/>`?

Reply:
and a badge with `updated` ? ::)

Reply:
The badge is already there  ? 🙂
hmm but i thought that the props table contains only props which is related to `<Table>` only,  not `<Table.Column>`